### PR TITLE
fix: remove id command

### PR DESCRIPTION
**Pull request by mafewtm**
## Description

Remove the ID Command in favor of having it in qbx_scoreboard

## Checklist

- [x] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [x] My code fits the style guidelines.
- [x] My PR fits the contribution guidelines.

### CODE REVIEW

File: server/main.lua [hunk 1]
```diff
@@ -2,12 +2,6 @@ RegisterNetEvent('tackle:server:TacklePlayer', function(playerId)
     TriggerClientEvent('tackle:client:GetTackled', playerId)
 end)
 
-lib.addCommand('id', {
-    help = 'Check Your ID #',
-}, function(source)
-    TriggerClientEvent('QBCore:Notify', source,  'ID: '..source)
-end)
-
```
Comment (Manason) [inline server/main.lua:10]: What does this have to do with qbx_socreboard?

**Review by Manason — CHANGES_REQUESTED** [review]

File: server/main.lua (hunk 1 above)
Comment (mafewtm) [reply server/main.lua:10]: Figured I'd put it in the scoreboard where you see your ID when you press HOME anyway instead of just leaving it in smallresources. If you know a better place to put it, we can move it.
